lxl-web/src/routes/(app)/[[lang=lang]]/find/+page.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+	import { page } from '$app/stores';
+	import DecoratedData from '$lib/components/DecoratedData.svelte';
+
+	export let data;
+
+	$: q = $page.url.searchParams.get('q');
+</script>
+
+<div class="m-3">
+	<h1>{$page.data.t('search.result_info', { q: `${q}` })}</h1>
+	<ul>
+		{#each data.searchResult.mapping as mapping}
+			<li>
+				<DecoratedData data={mapping.display} />
+				{#if 'up' in mapping}
+					<a class="underline" href={mapping.up['@id']}>x</a>
+				{/if}
+			</li>
+		{/each}
+	</ul>
+	<br />
+	<br />
+	<ul>
+		{#each data.searchResult.items as item (item['@id'])}
+			href={item.fnurgel}>{item['@id']}
```

Comment:
...both `fnurgel` and `@id` are undefined here.

Reply:
yep, look like some leftovers from the removed link